[android][CI] Problem with building Android Beta

Tried 2 times, no luck.
@biodranik @AndrewShkrob Is it possible with the recent CI changes with symbols/apk/debuginfo?

```
> Task :app:generateCrashlyticsSymbolFileGoogleBeta
File system event consumer thread stopped due to exception
java.lang.NullPointerException: Cannot invoke "Object.hashCode()" because "key" is null
> Task :app:lintVitalAnalyzeGoogleBeta FAILED
	at java.base/java.util.concurrent.ConcurrentHashMap.get(ConcurrentHashMap.java:936)
	at org.gradle.internal.watch.registry.impl.DefaultFileWatcherProbeRegistry.triggerWatchProbe(DefaultFileWatcherProbeRegistry.java:93)
	at org.gradle.internal.watch.registry.impl.AbstractFileWatcherUpdater.triggerWatchProbe(AbstractFileWatcherUpdater.java:132)
	at org.gradle.internal.watch.registry.impl.DefaultFileWatcherRegistry$1.handleChangeEvent(DefaultFileWatcherRegistry.java:82)
	at org.gradle.fileevents.internal.AbstractFileEventFunctions$ChangeEvent.handleEvent(AbstractFileEventFunctions.java:244)
	at org.gradle.internal.watch.registry.impl.DefaultFileWatcherRegistry.lambda$createAndStartEventConsumerThread$0(DefaultFileWatcherRegistry.java:78)
	at java.base/java.lang.Thread.run(Thread.java:840)
java.lang.OutOfMemoryError: Java heap space
	at org.gradle.fileevents.internal.AbstractNativeFileEventFunctions$NativeFileWatcher.executeRunLoop0(Native Method)
	at org.gradle.fileevents.internal.AbstractNativeFileEventFunctions$NativeFileWatcher.executeRunLoop(AbstractNativeFileEventFunctions.java:32)
	at org.gradle.fileevents.internal.AbstractFileEventFunctions$AbstractFileWatcher$1.run(AbstractFileEventFunctions.java:154)
> Task :app:uploadCrashlyticsSymbolFileGoogleBeta
> Task :app:uploadCrashlyticsMappingFileGoogleBeta

[Incubating] Problems report is available at: file:///home/runner/work/organicmaps/organicmaps/android/build/reports/problems/problems-report.html

FAILURE: Build failed with an exception.

* What went wrong:
Java heap space

* Try:
> Run with --stacktrace option to get the stack trace.
> Run with --info or --debug option to get more log output.
> Run with --scan to get full insights.
> Get more help at https://help.gradle.org./

BUILD FAILED in 18m 58s
```

--- a/android/gradle.properties
+++ b/android/gradle.properties
@@ -3,7 +3,7 @@ propTargetSdkVersion=36
 propCompileSdkVersion=36
 
 org.gradle.caching=true
-org.gradle.jvmargs=-Xmx1024m -Xms256m
+org.gradle.jvmargs=-Xmx2048m -Xms512m
 android.useAndroidX=true
 android.native.buildOutput=verbose
 android.nonTransitiveRClass=true

